types/openapi-factory/openapi-factory-tests.ts

```diff
@@ -0,0 +1,29 @@
+import * as OpenApiFactory from 'openapi-factory';
```

Comment:
I think this should be `import OpenApiFactory = require('openapi-factory')` to match the CommonJS export.

Reply:
Changed it